package.json

```diff
@@ -32,12 +32,14 @@
     "url": "https://github.com/buildo/react-placeholder/issues"
   },
   "homepage": "https://github.com/buildo/react-placeholder",
-  "dependencies": {},
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "lodash.omit": "^4.5.0"
```

Comment:
I know these dependencies are tiny, but you can probably skip both with

- const props = omit(this.props, ['children', 'ready', 'firstLaunchOnly', 'type']);
- const { children, ready, firstLaunchOnly, type, ...props } = this.props;

and simply joining strings instead of using `classnames` (which doesn't seem to be used in any interesting way)